samples/Eternal.fs

```diff
@@ -4,18 +4,17 @@ open System
 open Microsoft.Azure.WebJobs
 open DurableFunctions.FSharp
 
-let printTime = 
-    fun (d: DateTime) -> sprintf "Printing at %s!" (d.ToShortTimeString())
-    |> Activity.define "PrintTime"
+let printTime =     
+    Activity.define "PrintTime" (fun (d: DateTime) -> sprintf "Printing at %s!" (d.ToShortTimeString()))
 
 let workflow = orchestrator {
-    let! s = Activity.call printTime DateTime.Now
+    let! (s:string) = Activity.call printTime DateTime.Now
     do! Orchestrator.delay (TimeSpan.FromSeconds 5.0)
     return if s.Contains "00" then Stop else ContinueAsNew ()
 }
 
 let workflowWithParam delay = orchestrator {
-    let! s = Activity.call printTime DateTime.Now
+    let! (s:string) = Activity.call printTime DateTime.Now
```

Comment:
This is really weird indeed. I can't see why type inference is broken... Almost seems like a bug in the compiler. Maybe we should change the example not to use `.Contains` to avoid that.